README.md

```diff
@@ -1 +1,134 @@
-# art-gallery
\ No newline at end of file
+# art-gallery
+
+[You can visit our site from this link]([https://art-gallery-v1.herokuapp.com/])
+
+## Team Leader
+
+- Rana Obeid.
+
+## Team Members
+
+- Lina Ebeid.
+- Mariam Isa.
+- Muhammad Abdulhadi.
+- Alaa Abu Swaireh.
+
+## Summary
+
+Art gallery is an Arab web application which allows artists to view their arts, each artist has a profile to present his/her arts so customers can buy their paintings.
+
+## Challenge
+
+The inability of painters to sell their paintings and the difficulty in communicating with customers.
+
+## Solution
+
+Our site provides artists with the ability to sell their paintings at reasonable prices and the ability to communicate with the customer appropriately.
+
+## User Stories
+
+### As an artist
+
+- I can sign up / login and have my own profile
+- I can edit my profile data
+- I can upload my paintings in my profile
+
+### As an user
+
+- I can sign up / login and have an account
+- I can Check artworks for different artists
+- I can Search and filter art works
+- I can buy paintings and add it to my cart
+- I can Checkout from cart
+- I can add a review for the artist
+
+### As an admin
+
+- I can delete a artist/user/painting
+
```

Comment:
++ I can see all artist and  switch activate artist 
++ I can export all artist to excel file
++ I can see my budget on dashboard